applications/vessel-history/src/index.css

```diff
@@ -1,13 +1,20 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
-}
+:root {
+  --primary-green-rgb: 22, 137, 63; /* rgb(22, 63, 137) */
+  --color-primary-green: rgb(var(--primary-green-rgb));
+  --color-secondary-green: rgba(var(--primary-green-rgb), var(--opacity-secondary));
+  --color-terthiary-green: rgba(var(--primary-green-rgb), var(--opacity-terthiary));
+  --color-primary-green-hover: rgba(var(--primary-green-rgb), var(--opacity-hover));
+  --color-primary-green-border: rgba(var(--primary-green-rgb), var(--opacity-border));
+}
+
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
+    'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
```

Comment:
I like this, do you think worth it to move to the ui-components base styles ?

Reply:
Sure we can do it in a separate PR, WDYT @j8seangel ?

Reply:
Dale!